customs.js

```diff
@@ -55,6 +57,13 @@
       }
     )
     .then(
+      (function (result) {
+        if (result.lockout) {
+          return this.db.lockAccount(account)
```

Comment:
More broadly, the goal here is be able to decouple the merging code with who this is enabled for it production. Some options for who it's enabled for:

* everyone, behind a feature flag we could revert if it went horribly wrong
* mozilla.com emails
* randomly trigger it x% of the time it could be triggered

Regardless, as @rfk mentions, we need a dashboard to see how often it's getting triggered in prod and how often users are recovering. Ideally, we could match those up, and visualize:

* what percentage of accounts are locked
* number of new locked accounts
* number of newly unlocked accounts.